assets/js/components/wp-dashboard/WPDashboardSessionDuration.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * WPDashboardSessionDuration component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { prepareSecondsForDisplay } from '../../util';
+import DataBlock from '../data-block';
+import { STORE_NAME as MODULES_ANALYTICS } from '../../modules/analytics/datastore/constants';
+import { STORE_NAME as CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import { overviewReportDataDefaults } from '../../modules/analytics/util';
+import PreviewBlock from '../PreviewBlock';
+import ReportError from '../ReportError';
+import calculateOverviewData from '../../modules/analytics/util/calculateOverviewData';
+const { useSelect } = Data;
+
+const WPDashboardSessionDuration = () => {
+	const dateRange = useSelect( ( select ) => select( CORE_USER ).getDateRange() );
+
+	const args = {
+		dateRange,
+		...overviewReportDataDefaults,
+	};
```

Comment:
I don't think we should be using these old args anymore (see #2236 , #2237) but that may be outside of the scope of this ticket.

@felixarntz what do you think?

Reply:
@kostyalmm See above, this should use `getDateRangeDates` for passing the dates and also manually pass the other arguments instead of relying on the legacy `overviewReportDataDefaults`. You can probably copy most of that from https://github.com/google/site-kit-wp/blob/develop/assets/js/components/adminbar/AdminBarSessions.js, except that the expression should be `ga:avgSessionDuration` and the alias `Average Session Duration`.